erpnext/setup/workspace/erpnext_settings/erpnext_settings.json

```
@@ -109,6 +110,12 @@
    "label": "Domain Settings",
    "link_to": "Domain Settings",
    "type": "DocType"
+  },
+  {
+   "doc_view": "",
```

```suggestion
   "doc_view": "",
   "icon": "retail",
```

Add icon for consistency.